presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeWindowQueries.java

```diff
@@ -191,13 +191,18 @@ public void testLag()
     }
 
     @Test
-    public void testOverlappingPartitionAndSortingKeys()
+    public void testOverlappingPartitionAndSortingKeys1()
```

Comment:
TBH all of these lines should be separate tests. One test per *unit* test is both how unit testing is supposed to work, and definitely helps with debugging when something goes wrong.